contrib/gitian-build.py

```diff
@@ -51,7 +51,7 @@ def build():
     os.chdir('gitian-builder')
     os.makedirs('inputs', exist_ok=True)
 
-    subprocess.check_call(['wget', '-O' 'osslsigncode-2.0.tar.gz' '-N', '-P', 'inputs', 'https://github.com/mtrojnar/osslsigncode/archive/2.0.tar.gz'])
+    subprocess.check_call(['wget', '-O', 'inputs/osslsigncode-2.0.tar.gz', 'https://github.com/mtrojnar/osslsigncode/archive/2.0.tar.gz'])
```

Comment:
This is not just a typo but an actual bug-fix, right?

Reply:
right. a bug caused by typo?

Reply:
possibly, it's not really useful to guess
usually descriptions such as "typo" are used for harmless mistypings in comments
in any case if you fix actual problems it's better to say so in the PR title so that it's taken more seriously